packages/client/src/client.js

```diff
@@ -608,6 +612,36 @@ export class PercyClient {
     });
   }
 
+  // private function
+  #formatLogErrors(errorLogs) {
```

Comment:
Shift it to utils instead can be used at other places as well

Reply:
Shifted..., but it's a very specific function for formatting error logs. Don't know if it will work for other cases, as the payload may be different.